src/tests/utils.test.ts

```diff
@@ -11,3 +14,101 @@ describe("Test isTransparent", () => {
     expect(utils.isTransparent("#ced4da")).toEqual(false);
   });
 });
+
+describe("exportToSvg", () => {
+  const ELEMENT_HEIGHT = 100;
+  const ELEMENT_WIDTH = 100;
+  const ELEMENTS = [
+    { ...diamondFixture, height: ELEMENT_HEIGHT, width: ELEMENT_WIDTH },
+    { ...ellipseFixture, height: ELEMENT_HEIGHT, width: ELEMENT_WIDTH },
+  ] as NonDeletedExcalidrawElement[];
+
+  it("with default arguments", () => {
+    const svgElement = exportUtils.exportToSvg(ELEMENTS, {
+      exportBackground: false,
+      viewBackgroundColor: "#ffffff",
+      shouldAddWatermark: false,
+    });
+
+    expect(svgElement).toMatchSnapshot();
+  });
+
+  it("with background color", () => {
+    const BACKGROUND_COLOR = "#abcdef";
+
+    const svgElement = exportUtils.exportToSvg(ELEMENTS, {
+      exportBackground: true,
+      viewBackgroundColor: BACKGROUND_COLOR,
+      shouldAddWatermark: false,
+    });
+
+    expect(svgElement.querySelector("rect")).toHaveAttribute(
+      "fill",
+      BACKGROUND_COLOR,
+    );
+  });
+
+  it("with watermark", () => {
+    const svgElement = exportUtils.exportToSvg(ELEMENTS, {
+      exportBackground: false,
+      viewBackgroundColor: "#ffffff",
+      shouldAddWatermark: true,
+    });
+
+    expect(svgElement.querySelector("text")?.textContent).toMatchInlineSnapshot(
+      `"Made with Excalidraw"`,
+    );
+  });
+
+  it("with dark mode", () => {
+    const svgElement = exportUtils.exportToSvg(ELEMENTS, {
+      exportBackground: false,
+      viewBackgroundColor: "#ffffff",
+      exportWithDarkMode: true,
+      shouldAddWatermark: false,
```

Comment:
can we have some default args above so we can just spread over it instead of passing everytime ?